### ISSUE TITLE
[Adaptive Cards> InputsWithValidations.json]: Placeholder text is not defined for 'Text input with an inline action with no icon' edit field.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8187]
### Target Platforms

UWP

### SDK Version

3.0.2210.26001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25267.1000)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2210.26001

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to the 'InputsWithValidations.json' through tab key and activate it.
3. Now, navigate to 'Text input with an inline action with no icon' edit field through tab key.
4. Observe the issue.

**Actual Result:**
For 'Text input with an inline action with no icon' edit field, placeholder text is not defined.

**Expected Result:**
For 'Text input with an inline action with no icon' edit field, proper placeholder text should be defined. Such as **'Please enter your text input'.**

**Notes:**
Same issue repro throughout for all the edit fields.

**User Impact:**
People with cognitive disability will get impacted as user will not be able to know the purpose of the edit field or will get confused that what type of data needs to enter.

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/104483005/209769012-25b49280-0a8f-4d48-bea4-c9830f36a155.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is now working as expected, hence closing this issue.

![image](https://github.com/user-attachments/assets/66647d96-54ca-42dc-a85b-495c1ae55db4)
